htdocs/expensereport/card.php

```diff
@@ -2757,7 +2757,7 @@
 	* 	Afficher : "Enregistrer" / "Modifier" / "Supprimer"
 	*/
 	if ($user->hasRight('expensereport', 'creer') && $object->status == ExpenseReport::STATUS_DRAFT) {
-		if (in_array($object->fk_user_author, $childids) || $user->hasRight('expensereport', 'writeall_advance')) {
+		if (in_array($object->fk_user_author, $childids) || $user->hasRight('expensereport', 'readall')) {
```

Comment:
The button is a button to modify so it should be a permission to write no to read.

Reply:
ok